controller/tests/server/update/update_tests.ml

```diff
@@ -219,9 +244,10 @@ let () =
            [
              Alcotest_lwt.test_case "Delete downloaded bundle on install error"
              `Quick (fun _ () -> Scenario.run delete_downloaded_bundle_on_err);
-
-             Alcotest_lwt.test_case "Sleep for a duration after error or check"
```

Comment:
Replacing the sleep test (which was checking side-effects in a rather hacky way) with two different tests that check that Update transitions into `Sleeping` state after errors, because the "error" states actually no longer exist as process states.